code/game/machinery/Sleeper.dm

```diff
@@ -34,19 +34,7 @@
 /obj/machinery/sleeper/examine(mob/user)
 	. = ..()
 	if(Adjacent(user))
-		. += "<span class='notice'>You can <b>Alt-Click</b> to eject the current occupant.</span>"
-
-/obj/machinery/sleeper/detailed_examine()
-	return "The sleeper allows you to clean the blood by means of dialysis, and to administer medication in a controlled environment.<br>\
-			<br>\
-			Click on your target then drag their sprite onto the sleeper to put them into it. Click the sleeper, with an empty hand, to open the menu. \
-			Click 'Start Dialysis' to begin filtering unwanted chemicals from the occupant's blood. The beaker contained will begin to fill with their \
-			contaminated blood, and will need to be emptied when full.<br>\
-			<br>\
-			You can also inject common medicines directly into their bloodstream.\
-			<br>\
-			Alt-Click the sleeper to remove them. You can enter the cell yourself by right clicking and selecting 'Enter Sleeper'. \
-			Note that you cannot control the sleeper while inside of it."
+		. += "<span class='notice'>You can <b>Alt-Click</b> to eject the current occupant. Click-drag someone to the sleeper to place them in it after a short delay.</span>"
```

Comment:
```suggestion
		. += "<span class='notice'><b>Alt-Click</b> it to eject the current occupant.<br>\
		<b>Click-drag</b> someone to the sleeper to place them in it after a short delay.</span>"
```

For consistency's sake, I think the command should always be the first word(s) of the line.

It is also better to split up multiple commands on multiple lines because people tend to not read multi-line sentences.